packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/ConnectionLine.tsx

```diff
@@ -61,7 +62,9 @@ export const ConnectionLine = memo(
     const { diagramDescription } = useDiagramDescription();
     const { setNodes, getEdges, getEdge } = useReactFlow<Node<NodeData>, Edge<EdgeData>>();
     const updateNodeInternals = useUpdateNodeInternals();
-    const [previousToNodeId, setPreviousToNodeId] = useState<String>(toNode?.id || '');
+    const [state, setState] = useState<ConnectionLineState>({
+      previousToNodeId: toNode?.id || null,
```

Comment:
Prefer `??` to `||` in such use case because you will refuse any falsy value even if they are valid. For example if toNode.id was equal to 0 then it would return null instead of 0.